### PR TITLE
add grafana for global-hub release-2.9

### PR DESCRIPTION
**Pull request by morvencao**
ref: https://issues.redhat.com/browse/ACM-6696

### CODE REVIEW

File: ci-operator/config/stolostron/grafana/stolostron-grafana-global-hub-release-2.9.yaml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    tag: base
+  stolostron_builder_go1.18-linux:
+    name: builder
+    namespace: stolostron
+    tag: go1.18-linux
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: stolostron
+    tag: go1.18-linux
```
Comment (clyang82) [inline ci-operator/config/stolostron/grafana/stolostron-grafana-global-hub-release-2.9.yaml:14]: we are using go1.19-linux now for grafana 9.5.6.

Comment (morvencao) [reply ci-operator/config/stolostron/grafana/stolostron-grafana-global-hub-release-2.9.yaml]: updated to golang 1.19

File: ci-operator/config/stolostron/grafana/stolostron-grafana-global-hub-1.0.yaml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+images:
+- dockerfile_path: Dockerfile.ocp
+  from: base
+  inputs:
+    stolostron_builder_go1.19-linux:
+      as:
+      - registry.ci.openshift.org/stolostron/builder:go1.19-linux
+  to: grafana-global-hub
+promotion:
+  name: "2.9"
```
Comment (clyang82) [inline ci-operator/config/stolostron/grafana/stolostron-grafana-global-hub-1.0.yaml:24]: maybe 1.0.0 make more sense?

Comment (morvencao) [reply ci-operator/config/stolostron/grafana/stolostron-grafana-global-hub-1.0.yaml]: nice catch, updated this to 1.0

**Review by clyang82 — APPROVED** [review]
LGTM

**Comment by morvencao** [comment]
/pj-rehearse pull-ci-stolostron-grafana-global-hub-release-2.9-sonar-pre-submit

**Comment by morvencao** [comment]
@dptp

**Comment by morvencao** [comment]
@smg247

**Pull request merged**